src/include/cell.h

```diff
@@ -255,7 +243,7 @@ struct __wt_cell_unpack_delta_int {
     WT_CELL_UNPACK_KV key;
     WT_CELL_UNPACK_ADDR value;
 
-#define WT_DELTA_INT_IS_DELETE 0x01u
+#define WT_DELTA_INT_IS_VALID 0x01u
```

Comment:
Why we still have this?

Reply:
Fixed in 3ea4480f5894e0e5503aba112bb842dd9c2cb380